kill-halted-tasks/airflow-kill-halted-tasks.py

```diff
@@ -1,4 +1,19 @@
-from airflow.models import DAG, DagModel, DagRun, TaskInstance, settings
+from airflow import version
+from pkg_resources import parse_version
+
+if parse_version(version.version).base_version  == '1.10.5':
+   from airflow import settings
+   from airflow.models.dag import DAG, DagModel
+   from airflow.models.dagrun import DagRun
+   from airflow.models.taskinstance import TaskInstance
+elif parse_version(version.version).base_version == '1.10.3':
+   from airflow.models import DAG, DagModel, DagRun, TaskInstance, settings
+else:
+   from airflow import settings
+   from airflow.models.dag import DAG, DagModel
+   from airflow.models.dagrun import DagRun
+   from airflow.models.taskinstance import TaskInstance
+
```

Comment:
Same as above, this will work in both versions:
```
from airflow import settings
from airflow.models.dag import DAG, DagModel
from airflow.models.dagrun import DagRun
from airflow.models.taskinstance import TaskInstance
```